### DISCUSSION TITLE
Error computing bounding box on Android Webview

### DISCUSSION THREAD

**Discussion by anupam99** [vladmandic/face-api#189] [General]
HI, 
On an Android webview, i get this error :
```
Error: Box.constructor - expected box to be IBoundingBox | IRect, instead have {"left":null,"top":null,"right":null,"bottom":null}
    at new e (face-api.js:5009:20450)
    at new il (face-api.js:5009:22653)
    at kl.extractBoxes (face-api.js:5009:85169)
    at async kl.detect (face-api.js:5009:83700)
    at async kl.locateFaces (face-api.js:5009:87123)
    at async Hp.then (face-api.js:5009:86325)
```
i get similar error on the original library.
The models are being loaded from file:/// , and they load without any error, so i presume, error is due to something else.
Any ideas, Where should i start looking?
![image](https://github.com/vladmandic/face-api/assets/5384775/e4f1545f-b4a8-4c69-9b8b-6a2ce3fb2930)

**Comment by vladmandic** [comment]
i don't think face-api is compatible with webview, too many direct dom access points.
you could try newer human library.
